packages/chainspec/LICENSE

```diff
@@ -0,0 +1,201 @@
+                              Apache License
```

Comment:
Package created so there is a common place where the chainspecs can be held. Currently these are actually included in the client. Putting it here (and they don't update frequently) has the added benefit that we can actually use actual data in tests, for instance to calculate the trie-roots and then compare against known.